test/gateway/core.js

```diff
@@ -85,19 +124,19 @@ contract('Core', function (accounts) {
         });
 
         it('should not be able to commit state root of block height which is equal to latest block height', async () => {
-            await utils.expectThrow(core.commitStateRoot(blockHeight, stateRoot, {from: worker}));
+            await utils.expectThrow(core.commitStateRoot(blockHeight, stateRoot, { from: worker }));
```

Comment:
In the tests all the `it` looks interdependent. 
Should we consider to split this file into the function-specific test? (In this ticket or create a new ticket? Please suggest.)

Reply:
I would prefer a separate ticket to clean up the tests.

Reply:
Created a ticket for this #535